data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -190,6 +190,29 @@
         <addPrimaryKey columnNames="transaction_id, file_checksum"
                        constraintName="utxo_transaction_cpk_pkey"
                        tableName="utxo_transaction_cpk"/>
+
+        <comment>
+            Keep utxo_group_parameters's relevant fields in sync with vnode-vault-creation-v1.0.xml's vnode_group_parameters.
+        </comment>
+
+        <createTable tableName="utxo_group_parameters">
```

Comment:
Do we need a timestamp/created column like we have in most of our other tables?

Reply:
For consistency we could add. But the date itself can reflect only when we saved the record.
If there  are any dates inside the parameters, we won't be consistent with them.
Also the records coming from back chain resolution will have different dates in the different databases.  (Or we need to spend some time to stick the first date to the record when we send it forth and back.)

`vnode_group_parameters` which is roughly the same have an epoch field.

Reply:
I think a timestamp column would be good, and if it's just for as-of queries. On the other hand maybe not essential as we don't expect this to change a lot, and I can't foresee paged queries of group paramters.

Reply:
OK. I have just added it.